app_flutter/lib/state/flutter_build.dart

```diff
@@ -46,6 +49,8 @@ class FlutterBuildState extends ChangeNotifier {
   /// Request the latest [statuses] from [CocoonService].
   void _fetchBuildStatusUpdate() async {
     statuses = await _cocoonService.fetchCommitStatuses();
```

Comment:
These requests don't need to be made serially:

```
await Future.wait([
  _cocoonService.fetchCommitStatuses().then((commitStatus) => statuses = commitStatus),
  _cocoonService.fetchTreeBuildStatus().then((buildStatus) => statuses = buildStatus),
]);
```

Reply:
modulo types and formatting

Reply:
Thank you for sharing! This is a nice way of handling it.